metacat-connector-mysql/src/main/java/com/netflix/metacat/connector/mysql/MySqlConnectorTableService.java

```diff
@@ -0,0 +1,65 @@
+package com.netflix.metacat.connector.mysql;
+
+import com.google.inject.Inject;
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.server.connectors.model.AuditInfo;
+import com.netflix.metacat.common.server.connectors.model.TableInfo;
+import com.netflix.metacat.connector.jdbc.JdbcExceptionMapper;
+import com.netflix.metacat.connector.jdbc.JdbcTypeConverter;
+import com.netflix.metacat.connector.jdbc.services.JdbcConnectorTableService;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+
+/**
+ * Mysql table service implementation.
+ *
+ * @author amajumdar
+ * @since 1.2.0
+ */
+public class MySqlConnectorTableService extends JdbcConnectorTableService {
+
+    private static final String SQL_GET_AUDIT_INFO
+        = "select create_time, update_time from information_schema.tables where table_schema=? and table_name=?";
+    /**
+     * Constructor.
+     *
+     * @param dataSource      the datasource to use to connect to the database
+     * @param typeConverter   The type converter to use from the SQL type to Metacat canonical type
+     * @param exceptionMapper The exception mapper to use
+     */
+    @Inject
+    public MySqlConnectorTableService(
+        final DataSource dataSource,
+        final JdbcTypeConverter typeConverter,
+        final JdbcExceptionMapper exceptionMapper) {
+        super(dataSource, typeConverter, exceptionMapper);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected void setTableInfoDetails(final Connection connection, final TableInfo tableInfo) throws SQLException {
+        final QualifiedName tableName = tableInfo.getName();
```

Comment:
Should we name it as setTableAuditInfo()?

Reply:
I kept it more generic so that implementors of this can update any info in the table.